core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -138,6 +138,7 @@ class CivilizationInfo {
     fun isMajorCiv() = nation.isMajorCiv()
     fun isAlive(): Boolean = !isDefeated()
     fun hasEverBeenFriendWith(otherCiv: CivilizationInfo): Boolean = getDiplomacyManager(otherCiv).everBeenFriends()
+    fun hasMetCivTerritory(otherCiv: CivilizationInfo) = exploredTiles.any { gameInfo.tileMap[it].getOwner() == otherCiv }
```

Comment:
Disagree with this - instead of checking territory, we should check if the civs themselves have met.
Not only is this considerably faster, it also includes the option that they met through trade

Reply:
I know.. I would have implemented like you suggest, but in the original civ5 this is the requirement city states ask for. 
What to do? If you want to stick to the civ5 at 100% this is how it should be.